specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/stable/2023-03-15/examples/AssessmentProjectsOperations_Create_MaximumSet_Gen.json

```diff
@@ -0,0 +1,144 @@
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "createdTimestamp": "2023-09-15T07:22:18.588Z",
+          "updatedTimestamp": "2023-09-15T07:22:18.588Z",
+          "serviceEndpoint": "https://asmsrv.sea.test.migration.windowsazure.com/",
+          "assessmentSolutionId": "/subscriptions/4bd2aa0f-2bd2-4d67-91a8-5a4533d58600/resourceGroups/sakanwar/providers/Microsoft.Storage/storageAccounts/sakanwar1204usa",
+          "projectStatus": "Active",
+          "customerWorkspaceId": null,
+          "customerWorkspaceLocation": null,
+          "publicNetworkAccess": "Disabled",
+          "privateEndpointConnections": [
+            {
+              "properties": {
+                "privateEndpoint": {
+                  "id": "/subscriptions/31be0ff4-c932-4cb3-8efc-efa411d79280/resourceGroups/PrivLink-SelfHost/providers/Microsoft.Network/privateEndpoints/sakanwar1204project1634pe"
+                },
+                "privateLinkServiceConnectionState": {
+                  "status": "Approved",
+                  "description": null,
+                  "actionsRequired": ""
+                },
+                "provisioningState": "Succeeded"
+              },
+              "id": "/subscriptions/4bd2aa0f-2bd2-4d67-91a8-5a4533d58600/resourceGroups/sakanwar/providers/Microsoft.Migrate/assessmentprojects/sakanwar1204project/privateEndpointConnections/sakanwar1204project1634pe.bf42f8a1-09f5-4ee4-aea6-a019cc60f9d7",
+              "name": "sakanwar1204project1634pe.bf42f8a1-09f5-4ee4-aea6-a019cc60f9d7",
+              "type": "Microsoft.Migrate/assessmentprojects/privateEndpointConnections",
+              "systemData": {
+                "createdBy": "sakanwar",
+                "createdByType": "User",
+                "createdAt": "2023-09-15T07:22:18.589Z",
+                "lastModifiedBy": "sakanwar",
+                "lastModifiedByType": "User",
+                "lastModifiedAt": "2023-09-15T07:22:18.589Z"
+              }
+            }
+          ],
+          "customerStorageAccountArmId": "/subscriptions/4bd2aa0f-2bd2-4d67-91a8-5a4533d58600/resourceGroups/sakanwar/providers/Microsoft.Storage/storageAccounts/sakanwar1204usa",
+          "provisioningState": "Succeeded"
+        },
+        "tags": {
+          "Migrate Project": "sakanwar-PE-SEA"
+        },
+        "location": "southeastasia",
+        "id": "/subscriptions/4bd2aa0f-2bd2-4d67-91a8-5a4533d58600/resourceGroups/sakanwar/providers/Microsoft.Migrate/assessmentprojects/sakanwar1204project",
+        "name": "sakanwar1204project",
+        "type": "Microsoft.Migrate/assessmentprojects",
+        "systemData": {
+          "createdBy": "sakanwar",
+          "createdByType": "User",
+          "createdAt": "2023-09-15T07:22:18.589Z",
+          "lastModifiedBy": "sakanwar",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2023-09-15T07:22:18.589Z"
+        }
+      }
+    },
+    "201": {
+      "headers": {
```

Comment:
"Retry-After" header is missing in example.

Reply:
Our services dont send retry-after header. As per the API reference, retry header is an optional thing. While we will continue to improve our APIs in future and will work to take this up later, we will not be taking this now.
https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/async-api-reference.md#202-accepted-and-location-headers